aiida/orm/nodes/node.py

```diff
@@ -47,9 +49,16 @@ def __init__(self, fileobj, name):
     def _warn_if_not_entered(self, method):
         """Fire a warning if the object wrapper has not yet been entered."""
         if not self._was_entered:
-            msg = '`{}` used without context manager for {}. This will raise starting from `aiida-core==2.0.0`'.format(
-                method, self._name
-            )
+            msg = f'\n`Node.{method}` used outside context manager for {self._name}.\n' + \
+                  'Please use `with <node>.open(): ...` instead. ' + \
+                  'This will raise starting from `aiida-core==2.0.0`\n'
+            caller_frame = inspect.currentframe().f_back.f_back
+            caller_tb = inspect.getframeinfo(caller_frame)
+            msg += f'File {caller_tb.filename}, line {caller_tb.lineno} in module {caller_tb.function}'
```

Comment:
Not sure why, but I see the line with `File {caller_tb.filename} ..` twice
```
  /home/sph/code/aiida/env/dev/aiida-core/aiida/orm/nodes/node.py:62: AiidaDeprecationWarning: 
  `Node.read` used outside context manager for <ForceConstantsData: uuid: 6444b4db-4c69-484d-87d1-5ab6d38d8e46 (unstored)>.
  Please use `with <node>.open(): ...` instead. This will raise starting from `aiida-core==2.0.0`
  File /home/sph/code/aiida/env/dev/aiida-quantumespresso/aiida_quantumespresso/data/force_constants.py, line 22 in module set_file  File "/home/sph/code/aiida/env/dev/aiida-quantumespresso/aiida_quantumespresso/data/force_constants.py", line 22, in set_file
      content = handle.read().splitlines()
  
    warnings.warn(msg, AiidaDeprecationWarning)  # pylint: disable=no-member
```
Any idea why?

Reply:
Sorry, this didn't show up for me since I was working inside ipython, and so I thought I had to recreate this line manually.
I've removed it now; should be fixed.